stable/insights-agent/templates/aws-costs/rbac.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if and .Values.awscosts.enabled (not .Values.rbac.disabled) -}}
```

Comment:
I don't think we need any of this RBAC, since this job isn't accessing anything in the cluster

Reply:
done

Reply:
@rbren I added rback.yaml file otherwise the pode does not get created due permission.
`18s         Warning   FailedCreate        job/awscosts                                                                                                      job-controller               Error creating: pods "awscosts-" is forbidden: error looking up service account insights-agent-6gg8yo6c0h/insights-agent-6gg8yo6c0h-awscosts: serviceaccount "insights-agent-6gg8yo6c0h-awscosts" not found                                          48s          3       awscosts.169d1f1d0e6f9651`

Reply:
Ah you need to remove the `serviceAccount` field from the cronjob as well